ReactAndroid/src/main/java/com/facebook/react/modules/network/OkHttpClientProvider.java

```diff
@@ -43,6 +59,44 @@ public static OkHttpClient getOkHttpClient() {
     }
     return sClient;
   }
+
+  private static class EvictIdleConnectionsTask extends AsyncTask {
+    @Override
+    protected Object doInBackground(Object[] objects) {
+      for (OkHttpClient client: sClients) {
+        client.connectionPool().evictAll();
+        client.dispatcher().cancelAll();
```

Comment:
Should we perhaps cancel first, then evict? It seems like evicting then canceling would mean we end up with just those connections that *were* doing something sitting around in the pool, now idle. If we're going to be so rough as to cancel active requests, we should probably make sure those connections get evicted too.

Reply:
Hah, very good point!